react/src/components/molecules/ArrowNav/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {ArrowButton} from '../../../index';
+import './style.css';
+
+const ArrowNav = (props) => {
+  const sectionClasses = classNames({
+    'ma__arrow-nav': true,
+    [`ma__arrow-nav--${props.direction}`] : props.direction
+  });
+  const Element = props.href ? 'a' : 'button';
+  const onClickCallback = (e) => {
+    if (typeof props.onClick === 'function') {
+      e.preventDefault();
+      props.onClick(e);
+    }
+  };
+  return (
+    <section className={sectionClasses}>
+      <ArrowButton {...props} />
+      <h2 className="ma__arrow-nav__title" data-label={ props.label }>
+        <span>{props.title}</span>
+      </h2>
+      <div className="ma__arrow-nav__link">
+        <a
+          href={props.href}
+          onClick={(e) => onClickCallback(e)}
+          className="js-clickable-link"
+          title={props.info}
+        >
+          {props.text}
```

Comment:
This doesn't appear to render to screen at all when testing this on the story.

Reply:
Interesting.. I am seeing it in my local

![image](https://user-images.githubusercontent.com/7987464/46878954-f7a6e800-ce01-11e8-9d00-14b6348f143e.png)